internal/commands/image/tag.go

```diff
@@ -45,7 +45,7 @@ func runTag(bundleStore store.BundleStore, srcAppImage, destAppImage string) err
 	if err != nil {
 		return err
 	}
-
+	srcRef.RepoDigest = ""
```

Comment:
If I understand that correctly, tagging a pulled app image means we drop its repo digest? So it won't be displayed in the `docker app image ls --digest` column?

Reply:
I might be wrong, but IIUC repo digest won't be the same if you push under a distinct tag. So keeing this "pulled digest" for a tagged image would be wrong

Reply:
I think it should be the same repo digest, as tagging is just pushing a new descriptor, pointing to the same manifest/manifest list, but the repo digest should be the digest of the pointed object, not the pointer. cc @eunomie / @thaJeztah any idea?